app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java

```diff
@@ -612,6 +617,31 @@ public boolean onKey(View v, int keyCode, KeyEvent event) {
                     //and then manage our fade timer
                     if (mFadeEnabled) startFadeTimer();
                 }
+
+                if ( (playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) && !mGuideVisible && keyCode != KeyEvent.KEYCODE_BACK ) {
+                    //do some stuff here
```

Comment:
This comment is not really descriptive of what is happening here. What is the purpose of this check on EVERY key press??

Reply:
Updated comment. I chose to implement this function here because i could not find any other mechanism for making this check. This function fetches the currently playing item, and if its changed since we first started playing this channel, update the currently playing item. 

At the moment the currently playing item does not update as the channel continues playing. So if you're tuned to it for 2 hours, the currently playing item is still stuck on the original item with no good way to update it. I think updating the item only really matters when you want to view the overlay information. So i chose this action to update it.

Any suggestion for a better trigger mechanism to update the currently playing item?